CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.5+2
+
+- fix #21 - handle other successful status code (from http spec) in iOS
```

Comment:
Could you also write that extra check for android or shall I?

Reply:
android already does the check or rather the okHttp itself does it with isSuccessful method